content/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions.md

```diff
@@ -324,6 +325,9 @@ jobs:
 
 {% endpowershell %}
 
+> [!WARNING]
```

Comment:
I think the gist is correct that you can't actually use an `env:` block to set an unmasked secret and thus this section should be changed to just say as much:

> [!IMPORTANT]
> Values set via an `env:` block in a workflow **will** be logged. Trying to mask in the step would be too late.
> Masking would have to be done beforehand or by providing a file with the data and then reading the value, masking, and setting the variable locally within the shell context.